### PR TITLE
Update structure of feature drift nb

### PR DESCRIPTION
**Pull request by matanper**
1. Change some of the headers
2. Change opening sentence
3. Change condition print

I believe this structure is more global and well suited to be the same across all the checks

### CODE REVIEW

File: docs/source/examples/checks/distribution/train_test_feature_drift.ipynb [hunk 1]
```diff
@@ -3,26 +3,28 @@
   {
```
Comment (ItayGabbay) [inline docs/source/examples/checks/distribution/train_test_feature_drift.ipynb:3]: <p>Please add the target=_blank</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (ItayGabbay) [inline docs/source/examples/checks/distribution/train_test_feature_drift.ipynb:3]: <p>remove the #</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Review by ItayGabbay — CHANGES_REQUESTED** [review]
some minor fixes in the notebook

File: docs/source/examples/checks/distribution/train_test_feature_drift.ipynb (hunk 1 above)
Comment (matanper) [reply docs/source/examples/checks/distribution/train_test_feature_drift.ipynb:3]: <p>I don't where it is coming from here, it doesn't exists in the NB</p>

Comment (ItayGabbay) [reply docs/source/examples/checks/distribution/train_test_feature_drift.ipynb:3]: seems like it's a bug in the reviewnb.

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**